addons/cert-manager/template/testgrid/k8s-docker.yaml

```diff
@@ -38,9 +38,12 @@
     sleep 30s
```

Comment:
there really should be a mechanism that waits in the install script rather than here

Reply:
I didn't want to change the 1.0.3 installer, but I could probably add the same wait mechanism there

Reply:
I've changed the 1.0.3 installer now